openshift/jenkins/jenkins.dc.yaml

```diff
@@ -0,0 +1,310 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+labels:
+  app: jenkins
+metadata:
+  name: jenkins
+parameters:
+  - name: NAME
+    value: jenkins-wps
+    required: true
+  - name: NAMESPACE
+    value: e1e498-tools
+    required: true
+  - description: ROUTE_HOST
+    displayName: ROUTE_HOST
+    name: ROUTE_HOST
+    required: true
+    value: "wps-jenkins.apps.silver.devops.gov.bc.ca"
+  - description: ROUTE_PATH
+    displayName: ROUTE_PATH
+    name: ROUTE_PATH
+    required: true
+    value: /
+  - description: Environment Name
+    displayName: ENV_NAME
+    name: ENV_NAME
+    required: true
+    value: prod
+  - description: Environment ID
+    displayName: ENV_ID
+    name: ENV_ID
+    required: true
+    value: prod
+  - name: CPU_REQUEST
+    description: Minimal CPU needed to run
+    displayName: CPU Request
+    value: 300m
+  - name: CPU_LIMIT
+    description: Maximum CPU allowed to use
+    displayName: CPU Limit
+    value: 2000m
+  - name: MEMORY_REQUEST
+    description: Minimal amount of memory needed to run
+    displayName: Memory Request
+    value: 2Gi
+  - name: MEMORY_LIMIT
+    description: Maximum amount of memory allowed to use
+    displayName: Memory Limit
+    value: 4Gi
+  - description: SLAVE_USER_PASSWORD
+    displayName: SLAVE_USER_PASSWORD
+    from: "[a-zA-Z0-9]{16}"
+    generate: expression
+    name: SLAVE_USER_PASSWORD
+  - description: GitHub Username (Same as github secret)
```

Comment:
What GitHub credential are we using btw?

Reply:
See dev-chatter